app/services/dropdown_picker_service.js

```diff
@@ -0,0 +1,38 @@
+const getPickerFormat = (items) => {
```

Comment:
@limkimsan consider refactor this method to:

const getLanguageChoices = (lang_codes) =>
  let languages = [];
  lang_codes.map((code) => {
    lang_choice = {label: lang_dict[code], value: code};
    languages.push(lang_choice)
  });
  return languages